### ISSUE TITLE
Extension freezes editor.

### ISSUE THREAD

**Issue by itzmanish** [formulahendry/vscode-auto-rename-tag#466]
See what exact happens when i load react file.
https://github.com/microsoft/vscode/issues/77636

**Comment by danawoodman** [comment]
Same issue for me. Took me a while to track it down but for me I had a string that had an HTML tag in it and the extension froze the editor. It seems to happen on large HTML files especially when the line length is somehwhat long

**Comment by itzmanish** [comment]
> Same issue for me. Took me a while to track it down but for me I had a string that had an HTML tag in it and the extension froze the editor. It seems to happen on large HTML files especially when the line length is somewhat long

Yes, exactly. This seem only problem when a large file is loaded. However for now i stopped using this extension.

**Comment by ozguncagri** [comment]
Same issue but it occurs on even small react components. Problem is probably not related with the lines of code.
